actionpack/lib/system_test_case.rb

```diff
@@ -0,0 +1,57 @@
+require "system_testing/test_helper"
+require "system_testing/driver_adapter"
+
+module Rails
+  # System tests are similar to Integration tests in that they incorporate multiple
+  # controllers and actions, but can be used to similate a real user experience.
+  # System tests are also known as Acceptance tests.
+  #
+  # To create a System Test in your application extend your test class from
+  # <tt>Rails::SystemTestCase</tt>. System tests use Capybara as a base and
+  # allows you to configure the driver. The default driver is
+  # <tt>RailsSeleniumDriver</tt> which provides Capybara with no-setup
+  # configuration of the Selenium Driver. If you prefer you can use the bare
+  # Selenium driver and set your own configuration.
+  #
+  # A system test looks like the following:
+  #
+  #   require 'test_helper'
+  #
+  #   class Users::CreateTest < Rails::SystemTestCase
+  #     def adding_a_new_user
+  #       visit users_path
+  #       click_on 'New User'
+  #
+  #       fill_in 'Name', with: 'Arya'
+  #       click_on 'Create User'
```

Comment:
Should we do anything to make the Capybara helpers work better with Rails' integration helpers?

I haven't used Capybara much, but from what I can remember you can't call `visit` and then use `assert_response :success` for instance.

Pointing this out because the system test inherits from `ActionDispatch::IntegrationTest` so users might think our built in assertions would work.